### ISSUE TITLE
how to specify browser option e.g. to use Safari Technology Preview

### ISSUE THREAD

**Issue by JoeyHuangTexture** [webdriverio/webdriverio#2173]
## The problem

how to specify browser option e.g. to use Safari Technology Preview

**Comment by christian-bromann** [comment]
Probably like this.
```js
{
    browserName: 'safari',
    safariOptions: {...}
}
```
Please join our  [![Gitter](https://badges.gitter.im/webdriverio/webdriverio.svg)](https://gitter.im/webdriverio/webdriverio?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) for these kind of questions. This is not necessary an issue.

**Comment by JoeyHuangTexture** [comment]
I set it as below but it did not work. It still brings up Safari, not Safari Technology Preview.

capabilities: [{
        browserName: 'safari',
        safariOptions:{'technologyPreview':'true'}
    }],

In Selenium web driver (https://www.npmjs.com/package/selenium-webdriver), I can bring Safari Technology Preview by

const {Channel, Options} = require('../safari');

var safariOptions = new Options() ;
	safariOptions.setTechnologyPreview(true);

var driver = new Builder()
    .forBrowser('safari')
    .setSafariOptions(safariOptions)
    .build()

**Comment by christian-bromann** [comment]
I checked their implementation, it should be
```js
{
    browserName: 'safari',
   'safari.options': {
        technologyPreview: true
    }
}
```

**Comment by JoeyHuangTexture** [comment]
still not work with such setting

{
    browserName: 'safari',
   'safari.options': {
        technologyPreview: true
    }
}

**Comment by JoeyHuangTexture** [comment]
set it in test file works well. Set it in wdio.conf.js did not work.

**Comment by osala-netquest** [comment]
Is there any news on that?

**Comment by shashidharus** [comment]
https://github.com/SeleniumHQ/selenium/issues/4537#issuecomment-343899123

**Comment by burg** [comment]
My understanding is that you need to be using Selenium Server >= 3.8.0.

**Comment by thephilwells** [comment]
> set it in test file works well. Set it in wdio.conf.js did not work.

@JoeyHuangTexture could you provide a code sample where you got this to work within a test? Still struggling with this..

**Comment by khyati2010** [comment]
hey guys, not getting any reference how to set capabilities in safari, tried couple of combinations any hard-core solution for this?   {
            browserName: 'safari',
            maxInstances: 1,
            safariOptions: { technologyPreview: true }
        }
I am not able to by-pass this screen while executing on safari.

<img width="1377" alt="Screenshot 2020-08-28 at 2 16 45 PM" src="https://user-images.githubusercontent.com/6421649/91542532-a2afe480-e93b-11ea-888a-501e14f70cb0.png">


These are the solutions I tried:-
 {
browserName: 'safari',
maxInstances: 1,
// "safariOptions": { technologyPreview: true }
} 
&
 also tried providing credentials in the URL like this - baseUrl: 'https://user:pass@test.get.com', 
still I am getting the pop up while executing the scripts.
I have allowed remote execution on safari & ran this command on my local as well khyati.sehgal$ /usr/bin/safaridriver --enable, do I need to put some safarioptions in the config?
